src/main/drivers/pwm_mapping.c

```diff
@@ -228,6 +228,20 @@ static void timerHardwareOverride(timerHardware_t * timer) {
     }
 }
 
+bool check_pwm_assigned_to_motor_or_servo(void)
```

Comment:
Please rename this functions to `camelCase`

Reply:
Fixed. And  will delete this function once #9288 is merged